Source/UI/Validator/PhoneNumberValidator.swift

```diff
@@ -1,12 +1,13 @@
 import Foundation
 
 class PhoneNumberValidator: Validator {
-    func validate(text: String?) -> Bool {
-        isEmpty(text: text)
-    }
+    private let maxDigitLength = 25
+    private let minDigitLength = 6
 
-    private func isEmpty(text: String?) -> Bool {
-        text?.isEmpty ?? false
+    func validate(text: String?) -> Bool {
+        guard let text = text else { return true }
+        let number = text.components(separatedBy: CharacterSet.decimalDigits.inverted).joined()
+        return number.isEmpty || number.count < 6 || number.count > 25
```

Comment:
you can also leave a comment here refer : https://api-reference.checkout.com/#operation/requestAToken!path=0/phone&t=request